### DISCUSSION TITLE
Solving an issue

### DISCUSSION THREAD

**Discussion by mcNets** [unoplatform/uno#10189] [Q&A]
I would like to create a new PR related to [this](https://github.com/unoplatform/uno/issues/10110) issue.

@jeromelaban I think it can be solved by setting the Timelie.State to Stopped at the begining of OnCompleted method of Timeline class, before invoking the _completedHandlers list. I've tried using one of the existing sample apps for Skia and it works, but I can't check all platforms. I had a IOS developer license but I don't have any device right now.

I've made a slightly modification to the ColorAnimation_Background test to show the result.

**Comment by jeromelaban** [comment]
Thanks for the details! The best way to validate that this works is to [create a runtime test](https://platform.uno/docs/articles/uno-development/creating-runtime-tests.html?q=runtime), you can find [some of those here](https://github.com/unoplatform/uno/tree/master/src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Media_Animation). That way, it will be executed on CI on all platforms.
